storage/innobase/xtrabackup/test/t/history_on_server.sh

```diff
@@ -85,6 +85,15 @@ do
     check_for_value "$column" "N"
 done
 
+get_one_value "lock_time"
+lock_time_without_lock=$val
+
+if [ $val -lt 0 ];
+then	
+    vlog "Error: lock_time in history record invalid, expected > 0, got \"$val\""
+    exit 1
+fi
+
```

Comment:
Are we testing xtrabckup binary or innobakupex? whats the difference between two

Reply:
innobakupex is a wrapper on the xtrabackup binary.  which is available in 2.4x
 for 8.0, we are using directly xtrabackup